hazelcast/src/main/java/com/hazelcast/spi/eviction/EvictionPolicyComparator.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.spi.eviction;
```

Comment:
Should it be under package `com.hazelcast.eviction` rather than spi?

Reply:
It's SPI because the user can implement a custom policy, like the split-brain merge SPI.

Reply:
@ihsandemir I only followed what we did before and put it under spi, same as we did for split-brain merge.